### DISCUSSION TITLE
[Feature]: 挂件|热门文章统计-最多浏览和最多评论文章 || [Feature]: Widget | Popular article statistics - most viewed and most commented articles

### DISCUSSION THREAD

**Discussion by Doradx** [walinejs/waline#2078] [Ideas]
既然咱们waline已经提供了页面浏览量和评论数统计，那也可以给个API，提供站点文章统计信息，可自定义基于浏览量或评论数点赞。
这样的话，前端也可以自行适配，做成热点页面统计视图，方便读者知晓本站热门。

---
Now that waline has provided statistics on page views and comments, we can also provide an API to provide statistical information on site articles, which can be customized based on views or comments.
In this case, the front end can also adapt itself to make a statistical view of hot pages, so that readers can know the hot spots of this site.

**Comment by lizheming** [comment]
现在 Waline 并没有存储文章的标题等相关信息，内部去做可能暂时不太好实现。如果你懂开发的话可以开发个插件来实现这个事情，读取文章的浏览量和评论数还是很方便的。

---
Now Waline does not store relevant information such as the title of the article, and it may not be easy to implement internally. If you know how to develop, you can develop a plug-in to achieve this. It is very convenient to read the number of views and comments of the article.
